packages/gatsby/cache-dir/app.js

```diff
@@ -1,3 +1,6 @@
+// needed for fast refresh
+import "@gatsbyjs/webpack-hot-middleware/client"
```

Comment:
Previously refresh plugin would automatically fallback to importing client and using it. After https://github.com/pmmmwh/react-refresh-webpack-plugin/pull/337/files#diff-5425e895a25e9446e7307d31bc2cf26e1a77778571d76656d3b10876e9e48871 hot middleware client needs to be initialized elsewhere in the app and react-refresh will hook up to it (if available)

We can do it here as first import, or follow example and have it in our webpack entry ( https://github.com/pmmmwh/react-refresh-webpack-plugin/blob/5b0ef9b2ee2fb2f4323df3ec628c26b9c72bf0c7/examples/webpack-hot-middleware/webpack.config.js#L10-L12 )